src/command/eth/eth-command.ts

```diff
@@ -0,0 +1,19 @@
+import { RootCommand } from '../root-command'
+import { providers } from 'ethers'
+import { CommandLineError } from '../../utils/error'
+
+export class EthCommand extends RootCommand {
+  protected provider = new providers.JsonRpcProvider('http://localhost:9545')
```

Comment:
Does it need to be hardcoded? Maybe it's better to put it in env?

Reply:
you cannot change URL parameters in the CLI so it is always contant on the host.
Nevertheless I defined it in the constant.ts instead